src/v2/view-builder/views/IdentifierView.js

```diff
@@ -36,6 +36,25 @@ const Footer = BaseFooter.extend({
 });
 
 export default BaseView.extend({
+  initialize (options) {
+    // add callout for messages
+    this.messages = options.messages;
+    if (this.messages && this.messages.value.length) {
```

Comment:
I was thinking is to invoke showMessageCallout from BaseView. The idea is in BaseView, it renders callout whenever there are non-empty `messages`. what do you think?

Reply:
I see what you are saying now. Just moved this logic to BaseView.